### PR TITLE
Added Linux installation guide

### PR DESCRIPTION
**Pull request by GLAD-DE**
Info taken from [this issue](https://github.com/UnofficialCrusaderPatch/UnofficialCrusaderPatch/issues/868), [this anwser in particular](https://github.com/UnofficialCrusaderPatch/UnofficialCrusaderPatch/issues/868#issuecomment-881921762)

Closes issue #868

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,27 @@ This project is an unofficial patcher for [Firefly Studio's](https://fireflyworl
...
+
+After installing wine and winetricks, run the following in your terminal:
+
+```bash
+winetricks -q dotnet40
+```
+The GUI does not recognize hidden directories (any directory starting with "**.**") when you browse for files. You either move the game install location to a different location or create a symlink with:
+
+```bash
+ln -s ~/.steam/path/to/shc
```
Comment (gynt) [inline README.md:27]: Is this command complete? Does it work?

Comment (GLAD-DE) [reply README.md:27]: It is complete and it does work

Comment (GLAD-DE) [reply README.md:27]: The command is complete and it works
It creates a soft link to the directory with the name of the last directory. In this case the symlink would be called "shc"

Comment (GLAD-DE) [reply README.md:27]: To name the symlink one just has to add a name at the end. Example:
```bash
ln -s ~/.steam/path/to/stronghold stronghold
```

**Review by gynt — APPROVED** [review]
Thank you for the explanation!

**Pull request merged**